src/azure-cli/azure/cli/command_modules/acr/check_health.py

```diff
@@ -238,6 +238,13 @@ def _get_registry_status(login_server, registry_name, ignore_errors):
     from requests.exceptions import SSLError, RequestException
     from azure.cli.core.util import should_disable_connection_verify
 
+    # Get client's external ip, useful for firewall allowed ip list check. Note, Python doesn't have built-in support on this,
+    # so we use external endpoint as a best of effort
+    try:
+        print_pass("Your IP address: {}".format(requests.get('https://api.ipify.org').text))
```

Comment:
Using external site is not ideal, but I don't have other better options. I am contacting AppService team to see whether https://checkip.azurewebsites.net/ is owned by us. If yes, I will switch over